packages/x-license/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license",
-  "version": "7.0.0-beta.0",
+  "version": "7.0.0-beta.1",
```

Comment:
```suggestion
  "version": "7.0.0-beta.2",
```

Sync with pickers and data grid version?

Reply:
It was already out of sync. The current latest version is beta.0

Reply:
Yes, it was out of sync because there were no changes to release.
But since we release it now, I think we should sync it with the "main" version.